x-pack/plugins/ingest_pipelines/public/application/app.tsx

```diff
@@ -71,16 +71,32 @@ export const App: FunctionComponent = () => {
                 title={
                   <FormattedMessage
                     id="xpack.ingestPipelines.app.deniedPrivilegeTitle"
-                    defaultMessage="You're missing cluster privileges"
+                    defaultMessage="Cluster privileges required"
                   />
                 }
                 message={
                   <FormattedMessage
                     id="xpack.ingestPipelines.app.deniedPrivilegeDescription"
-                    defaultMessage="To use Ingest Pipelines, you must have {privilegesCount,
-                    plural, one {this cluster privilege} other {these cluster privileges}}: {missingPrivileges}."
+                    defaultMessage="You must have the {missingPrivileges} {privilegesCount,
+                    plural, one {privilege} other {privileges}} to access Ingest Node Pipelines."
                     values={{
-                      missingPrivileges: privilegesMissing.cluster!.join(', '),
+                      missingPrivileges: privilegesMissing
+                        .cluster!.map(privilege => <EuiCode>{privilege}</EuiCode>)
+                        .reduce((acc, currPrivilege, idx) => {
+                          if (idx === 0) {
+                            return currPrivilege;
+                          }
+                          return (
+                            <>
+                              {acc}{' '}
+                              <FormattedMessage
```

Comment:
I think it's worth checking with the i18n folks, but I don't think that we can depend on all languages following the English structure of using a word ("and" in the case of English) to join the last item onto a list. It might be safer to use the original pattern of simply joining on a comma.

Reply:
That's a good point. I reverted the change to the original implementation. This also follows the pattern we're using in Snapshot + Restore, so it will be consistent.